front/app/containers/IdeasNewSurveyPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -180,6 +186,12 @@ const IdeasNewSurveyForm = ({ project, phaseId }: Props) => {
     }
   };
 
+  // Re-check the page layout on desktop when the form changes
+  const onFormChange = () => {
+    !isSmallerThanPhone &&
+      setCheckCurrentLayout((currentValue) => !currentValue);
+  };
+
```

Comment:
Wasn't quite sure how best to tackle this one. If you have any other suggestion, just let me know! :)